pkgs/desktops/gnome-3/misc/nautilus-python/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, fetchFromGitLab
+, automake, autoconf, libtool, pkgconfig, which, gnome-common, gtk-doc, file
+, python3, python3Packages, nautilus }:
+
+stdenv.mkDerivation rec {
+  name = "nautilus-python-${version}";
+  version = "1.2.2";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "GNOME";
+    repo = "nautilus-python";
+    rev = "44224fac32d2d5284956b55ecd884fdfcb1bec4e";
```

Comment:
You could use `NAUTILUS_PYTHON_${stdenv.lib.replaceChars ["."] ["_"] version}` but it is better to use `fetchurl` with `mirror://gnome`:

https://github.com/NixOS/nixpkgs/blob/c3a1c0008baf467886806cd00430de7a9fe89d06/pkgs/desktops/gnome-3/core/nautilus/default.nix#L11-L14